pkg/resources/reconciling/ensure.go

```diff
@@ -34,6 +34,12 @@ import (
 	ctrlruntimeclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	// ResourceReconciliationPausedAnnotation is the name of the annotation to set to "true" to stop any
```

Comment:
We should clearly document that doing so will "void any kind of warranty" and that there is no support for systems that make use of this hack. I can already imagine the horrors of debugging for 3 days only to find a stray annotation somewhere. 🙈

Reply:
extended that comment with a note for that and also added that to the new development docs section